### PR TITLE
Resolves github #759 - removes altitude from storage

### PR DESCRIPTION
**Pull request by AdamLeonSmith**
#### Description:

Set altitude to 0 before storing, per issue report

#### Linked issues:

https://github.com/Path-Check/covid-safe-paths/issues/759

#### How to test:

Test on a device, copy the local storage database to a computer, analyse.

### CODE REVIEW

**Comment by AdamLeonSmith** [comment]
I don't see a way around the lint warning...

File: app/services/LocationService.js [hunk 1]
```diff
@@ -71,6 +71,9 @@ export class LocationData {
       let unixtimeUTC = Math.floor(location['time']);
       let unixtimeUTC_28daysAgo = unixtimeUTC - 60 * 60 * 24 * 1000 * 28;
 
+      // Remove altitude as not Required
+      let altitude = 0;
```
Comment (tstirrat) [inline app/services/LocationService.js:75]: The link warning is correct, this is never used anywhere.

Comment (tstirrat) [inline app/services/LocationService.js:75]: The link warning is correct, this is never used anywhere.

**Comment by AdamLeonSmith** [comment]
@tstirrat I have pulled the changes from develop and re-implemented.  I've tested the changes on Android but it is difficult to confirm, because of the secure storage.  Can I get any example code for logging the hex-encoded key?

**Comment by summetj** [comment]
> @tstirrat I have pulled the changes from develop and re-implemented. I've tested the changes on Android but it is difficult to confirm, because of the secure storage. Can I get any example code for logging the hex-encoded key?

Hi Adam, will the information here answer your question about extracting the key?
// https://pathcheck.atlassian.net/wiki/spaces/TEST/pages/50824879/Secure+Storage+Testing
// -or-
// https://gist.github.com/troach-sf/f257bb7b80e6dddd4f3bade81b7b1410

**Comment by tstirrat** [comment]
This seems good to merge, please merge `develop` and we can get it in.

**Pull request closed without merging**